internal/testing/e2e/licensemanager/license.go

```diff
@@ -0,0 +1,39 @@
+//NOTE - this code is copied directly from this file https://github.com/hashicorp/consul-enterprise/blob/6099c553634223a9a7ddf2263d90eeaef98c50e0/agent/licensemanager/loader.go#L24-L36
+// the package isn't being used directly as part of a conscious choice to limit using consul-enterprise as a direct dependency
+
+package licensemanager
```

Comment:
Since this is super Consul-specific and we really have no other place in our testing environment that requires reading Consul licenses for the time being, I think it'd be fine just dropping the `LoadLicense` function into an unexported helper in `internal/testing/e2e/consul.go`.

If we ever needed to do anything more complicated like live-reloading, etc. it'd make sense to move it into its own subpackage, but this is simple enough I think we should just keep it with the other bootstrapping code for spinning up the Consul test environment -- makes the code easier to track.